src/workloads/query/AggregationsOutput.yml

```diff
@@ -35,480 +21,184 @@ Clients:
-        OperationCommand:
-          Pipeline: [
-            {$addFields: {counter: {$add: [1, '$counter']}}},
-            {$merge: {into: {^Parameter: {Name: UnshardedCollOutput, Default: *UnshardedCollOutput1}}}}
-          ]
-    - Phase: {^PreprocessorFormatString: {format: "%d..%d", withArgs: [
-        {^NumExpr: {withExpression: "startsAt + 2 + numCombinations * 2 + 2", andValues: {startsAt: {^Parameter: {Name: StartsAt, Default: 2}}, numCombinations: {^Parameter: {Name: NumCombinations, Default: 1}}}}},
-        {^NumExpr: {withExpression: "startsAt + 2 + numCombinations * 3 + 1", andValues: {startsAt: {^Parameter: {Name: StartsAt, Default: 2}}, numCombinations: {^Parameter: {Name: NumCombinations, Default: 1}}}}}
-      ]}}
-      Nop: true
-    # TODO SERVER-84185: Enable this and remove the Nop that follows when sharded output collections are supported without DuplicateKey errors.
-    # - Duration: 20 seconds
-    #   Database: *Database
-    #   Collection: {^Parameter: {Name: "SrcCollection", Default: Collection0}}
-    #   Operations:
-    #   - OperationMetricsName: mergeToNewCollSharded
-    #     OperationName: aggregate
-    #     OperationCommand:
-    #       Pipeline: [
-    #         {$addFields: {counter: {$add: [1, '$counter']}}},
-    #         {$merge: {into: {^Parameter: {Name: ShardedCollOutput, Default: *ShardedCollOutput1}}}}
-    #       ]
-    - Nop: true
-    - Phase: {^PreprocessorFormatString: {format: "%d..%d", withArgs: [
-        {^NumExpr: {withExpression: "startsAt + 2 + numCombinations * 3 + 3", andValues: {startsAt: {^Parameter: {Name: StartsAt, Default: 2}}, numCombinations: {^Parameter: {Name: NumCombinations, Default: 1}}}}},
-        *NumPhases
-      ]}}
-      Nop: true
-Actors:
-- Name: Unsharded1KLoader
-  Type: MonotonicSingleLoader
-  Threads: 5
-  Phases:
-    OnlyActiveInPhases:
-      Active: [1]
-      NopInPhasesUpTo: *NumPhases
-      PhaseConfig:
-        Repeat: 1
-        Database: *Database
-        Collection: *Unsharded1K
-        DocumentCount: *NumDocs
-        BatchSize: 100
-        Document: *1KDoc
-- Name: Unsharded4KLoader
-  Type: MonotonicSingleLoader
-  Threads: 5
-  Phases:
-    OnlyActiveInPhases:
-      Active: [1]
-      NopInPhasesUpTo: *NumPhases
-      PhaseConfig:
-        Repeat: 1
-        Database: *Database
-        Collection: *Unsharded4K
-        DocumentCount: *NumDocs
-        BatchSize: 100
-        Document: *4KDoc
-- Name: Unsharded16KLoader
-  Type: MonotonicSingleLoader
-  Threads: 5
-  Phases:
-    OnlyActiveInPhases: 
-      Active: [1]
-      NopInPhasesUpTo: *NumPhases
-      PhaseConfig:
-        Repeat: 1
-        Database: *Database
-        Collection: *Unsharded16K
-        DocumentCount: *NumDocs
-        BatchSize: 100
-        Document: *16KDoc
+      OnlyActiveInPhases:
+        Active: [{^Parameter: {Name: "ActivePhase", Default: 0}}]
+        NopInPhasesUpTo: *NumPhases
+        PhaseConfig:
+          Repeat: 10
```

Comment:
Are we sure only doing 10 repetitions will be enough? The first one should be quite quick to execute and I would expect a lot of noise for the first metric with only 10 executions. The 10k limit would be fine since there's a lot of docs to process and the noise would be dwarfed by the processing.

Reply:
To verify the noisiness I'd look at the output of multiple remote executions running this. If we see that it isn't noisy there I'd be open to mark this as resolved.

Otherwise what you suggest (increasing the number of repetitions) is fine since we have to get the mean value somehow and the only solution is brute-force in this case.

Reply:
I've run a patch 4 times and the short operation metrics don't look super noisy when compared to the other ones. I'll increase the number of repetitions to 20 anyway. Does this sound good?

Reply:
Forgot to attach the results:
![Screenshot 2024-03-18 at 12 21 03](https://github.com/mongodb/genny/assets/9105982/c5c5c5e4-8d19-4b4d-9f6d-3cf5e135790b)

Reply:
SGTM